src/app/games/hive/tests/HiveMove.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { Coord } from 'src/app/jscaip/Coord';
+import { Player } from 'src/app/jscaip/Player';
+import { EncoderTestUtils } from 'src/app/utils/tests/Encoder.spec';
+import { HiveMove } from '../HiveMove';
+import { HivePiece } from '../HivePiece';
+
+describe('HiveMove', () => {
```

Comment:
ça manque de GWT, mopâté !

Reply:
dans les Move on faisait pas de GWT ? :/ Car chaque it a plein d'expect, ou bien on veut réellement faire « Given a drop move. When calling toString. Then it should be defined. » « Given a slide move. When calling toString. Then it should be defined » ? Le but des commentaires n'est-il pas d'ajouter de l'information au code, pas de noyer le code dans de la redondance ?

Reply:
Haha, c'plus une politesse pour l'homogénéité, mais te force pas en effet ! Ou juste un par review :D !

Reply:
j'me concentre sur le reste plutôt :p